src/app/home-page/home-logged-out/home.component.scss

```diff
@@ -28,10 +28,7 @@
 }
 
 button {
-  background-color: mat-color($dockstore-app-accent);
-  color: white;
-  padding: 1rem;
-  text-decoration: none;
+  padding: 0.3rem 1rem;
```

Comment:
what's so special about the buttons on the logged out homepage that it has different padding than all the buttons in the entire site?

Reply:
what about other pages? what padding are tehy using?

Reply:
Seems like mat-raised-buttons on other pages don't have padding (just the default padding from mat-raised-buttons)

Reply:
looks like the default is 0px and 16px. if you're going to add padding for buttons, add it globally (unless there's a good reason)

Reply:
Edit: removed the additional padding for buttons on the home page and about page because it didn't make that much of a difference (pics above for comparison). It still has the default 0px and 16px padding